src/v5_hal/firmware/src/nodes/LiftNode.cpp

```diff
@@ -1,17 +1,20 @@
 #include "nodes/LiftNode.h"
 
 LiftNode::LiftNode(NodeManager* node_manager, std::string handle_name, 
-        ControllerNode* controller, MotorNode* left_motor, 
+        ControllerNode* controller, pros::controller_digital_e_t upButton, 
+        pros::controller_digital_e_t downButton, MotorNode* left_motor, 
         MotorNode* right_motor, ADIDigitalInNode* bottom_limit_switch, 
         ADIDigitalInNode* top_limit_switch, ADIAnalogInNode* potentiometer) : 
         ILiftNode(node_manager, handle_name), 
         m_controller(controller),
+        m_upButton(upButton),
+        m_downButton(downButton),
         m_left_motor(left_motor),
         m_right_motor(right_motor),
         m_bottom_limit_switch(bottom_limit_switch),
         m_top_limit_switch(top_limit_switch),
         m_potentiometer(potentiometer),
-        m_lift_state(HOLDING),
+        m_lift_state(FULLY_UP),

```

Comment:
We likely want to have the starting state as "DOWN" or "HOLDING" (if we add that)